CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.1.0 [2022-06-03]
+
+_Breaking changes_
+
+- Removed the column `raw` from all the tables since the data included in the column was redundant. ([#6](https://github.com/turbot/steampipe-plugin-panos/pull/6))
```

Comment:
```suggestion
- Removed the column `raw` from all tables since the data included in the column was redundant. ([#6](https://github.com/turbot/steampipe-plugin-panos/pull/6))
```